src/content/en/2019/pwa.md

```diff
@@ -0,0 +1,208 @@
+---
+part_number: [II]
+chapter_number: [11]
+title: Progressive Web Apps
+authors: [tomayac, jeffposnick]
+reviewers: [hyperpress]
```

Comment:
```suggestion
reviewers: [hyperpress, ahmadawais]
```

Reply:
@rviscomi just finished reviewing the PWA chapter, figured I should help out while waiting for the Media chapter. I hope it's fine to add my name in the reviewers here.

Reply:
Absolutely, thank you for reviewing!